bin/deploy_k8s

```diff
@@ -68,6 +68,7 @@ main() {
   manifest=$("$(repo_directory)/bin/generate_kubo_manifest" "${bosh_env}" "${deployment_name}")
   stemcell_version=$(echo -n "${manifest}" | bosh-cli int - --path '/stemcells/0/version')
   upload_stemcell "${release_source}" "${platform}" "${stemcell_version}"
+  set_runtime_config
```

Comment:
It's set in response to a flag in the env config; made sense in our thinking during impl. Having this live in CI only makes the deployment confusing, as diagnostics should be configurable to run outside of our CI also.